### PR TITLE
Sree/updated bignumber.js -> bn.js

### PR DESCRIPTION
**Pull request by ssreelekhya**
updated bignumber.js -> bn.js

### CODE REVIEW

File: docs/api_abi-web3.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ async function main() {
 
 async function broadcastTx(txObject) {
   const gasLimit = await txObject.estimateGas();
-  const gasPrice = new BN(50).times(10 ** 9); // 50 Gwei
+  const gasPrice = new BN (50).mul(new BN (10).pow(new BN (9))); // 50 Gwei
```
Comment (ayobuenavista) [inline docs/api_abi-web3.md]: Some formatting fixes. Please fix to ```...new BN(10).pow(new BN(9)));`. Basically there's a space between BN and the parenthesis.

File: docs/integrations-feesharing.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ async function main() {
 
 async function broadcastTx(txObject) {
   const gasLimit = await txObject.estimateGas();
-  const gasPrice = new BN(50).times(10 ** 9); // 50 Gwei
+  const gasPrice = new BN (50).mul(new BN (10).pow(new BN (9))); // 50 Gwei
```
Comment (ayobuenavista) [inline docs/integrations-feesharing.md]: Same as above.

File: docs/integrations-feesharing.md [hunk 3]
```diff
@@ -270,7 +270,7 @@ async function main() {
 async function broadcastTx(txObject) {
   const gasLimit = await txObject.estimateGas();
   console.log(gasLimit);
-  const gasPrice = new BN(50).times(10 ** 9); // 50 Gwei, change to your preferred gas price
+  const gasPrice = new BN (50).mul(new BN (10).pow(new BN (9))); // 50 Gwei, change to your preferred gas price
```
Comment (ayobuenavista) [inline docs/integrations-feesharing.md]: Same as above.

File: docs/integrations-web3guide.md [hunk 4]
```diff
@@ -149,7 +149,7 @@ Note that we check that the gas price to be used does not exceed the maximum gas
 async function broadcastTx(from, to, txData, value, gasLimit) {
   let txCount = await web3.eth.getTransactionCount(USER_ADDRESS);
   //Method 1: Use a constant
-  let gasPrice = new BN(5).times(10 ** 9); //5 Gwei
+  let gasPrice = new BN (5).mul(new BN (10).pow(new BN (9))); ; //5 Gwei
```
Comment (ayobuenavista) [inline docs/integrations-web3guide.md]: Same as above.

File: docs/integrations-web3guide.md [hunk 5]
```diff
@@ -409,7 +409,7 @@ async function getGasLimit(source, dest, amount) {
 async function broadcastTx(from, to, txData, value, gasLimit) {
   let txCount = await web3.eth.getTransactionCount(USER_ADDRESS);
   //Method 1: Use a constant
-  let gasPrice = new BN(5).times(10 ** 9); //5 Gwei
+  let gasPrice = new BN (5).mul(new BN (10).pow(new BN (9))); //5 Gwei
```
Comment (ayobuenavista) [inline docs/integrations-web3guide.md]: Same as above.

File: docs/integrations-web3guide.md [hunk 6]
```diff
@@ -573,7 +573,7 @@ Note that we check that the gas price to be used does not exceed the maximum gas
...
```
Comment (ayobuenavista) [inline docs/integrations-web3guide.md]: Same as above.

File: docs/integrations-web3guide.md [hunk 7]
```diff
@@ -897,7 +897,7 @@ async function getGasLimit(source, dest, amount) {
...
```
Comment (ayobuenavista) [inline docs/integrations-web3guide.md]: Same as above.

File: docs/integrations-web3guide.md [hunk 8]
```diff
@@ -1078,7 +1078,7 @@ Note that we check that the gas price to be used does not exceed the maximum gas
...
```
Comment (ayobuenavista) [inline docs/integrations-web3guide.md]: Same as above.

File: docs/integrations-web3guide.md [hunk 9]
```diff
@@ -1402,7 +1402,7 @@ async function getGasLimit(source, dest, amount) {
...
```
Comment (ayobuenavista) [inline docs/integrations-web3guide.md]: Same as above.

File: docs/reserves-fedpricereserve.md [hunk 10]
```diff
@@ -274,7 +274,7 @@ async function main() {
       value: 0,
       data: txData,
       gas: 300000, //gasLimit
-      gasPrice: new BN(10).times(10 ** 9), //10 Gwei
+      gasPrice: new BN (10).mul(new BN (10).pow(new BN (9))), //10 Gwei
```
Comment (ayobuenavista) [inline docs/reserves-fedpricereserve.md]: Same as above.

**Comment by ssreelekhya** [comment]
changes made.

**Pull request merged**